CMakeLists.txt

```diff
@@ -137,6 +138,46 @@ else()
     message(STATUS "    LIBHWLOC_LIBRARIES = ${LIBHWLOC_LIBRARIES}")
```

Comment:
these messages could be moved below the if, or copy-pasted, or something 😉

Reply:
Done.

Reply:
thx!